data_models/data_refinery_models/models/organism.py

```diff
@@ -0,0 +1,121 @@
+import requests
+from xml.etree import ElementTree
+from django.db import models
+from data_refinery_models.models.base_models import TimeTrackedModel
+
+# Import and set logger
+import logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+NCBI_ROOT_URL = "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/"
+ESEARCH_URL = NCBI_ROOT_URL + "esearch.fcgi"
+EFETCH_URL = NCBI_ROOT_URL + "efetch.fcgi"
```

Comment:
I think that sounds reasonable for the organisms - if you think this is a better fit then go for it.

In terms of the URLs, I think that at some point we might want to get this information from another URL endpoint. Also, what if we have a saved XML file with the organism information we want and want to pass it to these methods and model? What parameters would we need to abstract to make this possible?